modules/server-node/examples/0-config.http

```diff
@@ -71,3 +73,11 @@ Content-Type: application/json
 {
   "index": 0
 }
+
+### CREATE DAVE
+POST {{daveUrl}}/node
```

Comment:
Revert this change please.

Reply:
yeah sorry, it was for testing. 
I reverted the examples to originals, and added an 'auction.http' for testing. It can be discarded